src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedParser.cs

```diff
@@ -480,12 +483,10 @@ private static string GetString(XElement parent, XName childName)
                                 Strings.Protocol_duplicateUri,
                                 nextUri));
                         }
-
-                        cacheKey = $"list_{relativeUri}_page{page}";
+                        page++;
```

Comment:
this is the only line that needs to be fixed, right. As this is escrow, please undo all the other refactoring.  DoRon wants to get in front of the QB and inserted before 6pm, if she approves.